util/flags.py

```diff
@@ -113,3 +113,21 @@ def create_flags():
     # Inference mode
 
     f.DEFINE_string('one_shot_infer', '', 'one-shot inference mode: specify a wav file and the script will load the checkpoint and perform inference on it.')
+
+    # Register validators for paths which require a file to be specified
+
+    f.register_validator('lm_binary_path',
+                         lambda value: os.path.isfile(value),
+                         message='The file pointed to by --lm_binary_path must exist and be readable.')
+
+    f.register_validator('lm_trie_path',
+                         lambda value: os.path.isfile(value),
+                         message='The file pointed to by --lm_trie_path must exist and be readable.')
+
+    f.register_validator('alphabet_config_path',
+                         lambda value: os.path.isfile(value),
+                         message='The file pointed to by --alphabet_config_path must exist and be readable.')
+
+    f.register_validator('one_shot_infer',
+                         lambda value: os.path.isfile(value),
```

Comment:
Seems like validators are always run, even for default values. So for optional flags you need to check that case, or it errors out whenever you don't specify `--one_shot_infer`. You can change the checker to be:

```python
lambda value: not value or os.path.isfile(value)
```